.storybook/config.js

```diff
@@ -3,6 +3,7 @@ import { configure } from '@storybook/react';
 function loadStories() {
   require('../stories');
   require('../stories/button');
+  require('../stories/notification');
```

Comment:
If you want, in your branch, you can just dynamically require everything inside /stories/*.js.

https://storybook.js.org/basics/writing-stories/#loading-stories-dynamically

:)

Reply:
will do from now on